pages/linux/vboxmanage.md

```diff
@@ -0,0 +1,29 @@
+# VBoxManage
+
+> VBoxManage is the command-line interface to VirtualBox.
+> With it, you can completely control VirtualBox from the command line.
+> Homepage: <https://www.virtualbox.org>.
```

Comment:
```suggestion
> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-intro>.
```
I'd say this is a better link (providing full and detailed information on the command). Also, let's use `More information` instead of `Homepage` as this is what pages should use.